docs/components/beacon/beacon-demo.js

```diff
@@ -70,4 +70,49 @@ import Util from '../../_util';
             },
         });
     }//vue-beacon-inline-styles
+
+    if (document.getElementById('vue-beacon-default-styles')) {
+        new Vue({
+            el: '#vue-beacon-default-styles',
+            data: {
+                beaconDefaultStyles: 'top: 42%; left: 24px; --hxBeacon: #d32f2f;',
+                isOpen: false,
+                isDismissed: false,
+            },
+            methods: {
+                onEvent: function () {
+                    this.isOpen = true;
+                    this.isDismissed = true;
+                },
+
+            },
+            computed: {
+                beaconSnippet: function () {
+                    if (this.isDismissed) {
+                        return '';
+                    } else {
+                        return `<hx-beacon style="${this.beaconDefaultStyles}"></hx-beacon>`;
```

Comment:
Please change to reflect the demo - `beaconOverrideStyles`.

Reply:
Changes have been done, Please review